src/main/java/org/reactome/release/downloaddirectory/GenerateGOAnnotationFile/MolecularFunctionAnnotationBuilder.java

```diff
@@ -4,9 +4,9 @@
 import org.apache.logging.log4j.Logger;
 import org.gk.model.GKInstance;
 import org.gk.model.ReactomeJavaConstants;
-import org.gk.schema.SchemaClass;
 
 import static org.reactome.release.downloaddirectory.GenerateGOAnnotationFile.GOAGeneratorConstants.*;
+import static org.reactome.release.downloaddirectory.GenerateGOAnnotationFile.GOAGeneratorUtilities.*;
```

Comment:
I replaced the * import with:

```
import static org.reactome.release.downloaddirectory.GenerateGOAnnotationFile.GOAGeneratorUtilities.getGOAnnotatableProteinsFromCatalystActivity;
import static org.reactome.release.downloaddirectory.GenerateGOAnnotationFile.GOAGeneratorUtilities.getAnyIssueForAnnotationDisqualification;
```

I think the import is justified for this case because the full qualification makes the method body they are called in difficult to read.